packages/next/src/server/lib/start-server.ts

```diff
@@ -266,8 +266,10 @@ export async function startServer(
       try {
         const cleanup = (code: number | null) => {
           debug('start-server process cleanup')
-          server.close()
-          process.exit(code ?? 0)
+          Log.info('Shutting down server...')
```

Comment:
Let's not add extra logging for now. We can add this in other PR if necessary for other visualizing purpose
```suggestion
```